lab-trent/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+let HTMLPlugin = require('html-webpack-plugin');
+let ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  entry: './app/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: 'build'

```

Comment:
This is also another break point for webpack 2.x. You need to absolute path to the build path..